### PR TITLE
Add `08-wasm` clients module

### PR DESCRIPTION
**Pull request by aeryz**
Add `08-wasm` clients to enable IBC wasm light clients: https://ibc.cosmos.network/v8/ibc/light-clients/wasm/integration/

Note that a separate wasmvm is being used for `ibc-wasm` to fully isolate `x/wasm` and `ibc-wasm` modules. Sharing VM's were [caused some issues](https://medium.com/@reecepbcups/juno-v19-post-mortum-969f251bffb7) for other people previously.

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -1296,3 +1328,16 @@ func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino
 
 	return paramsKeeper
 }
+
+// Capabilities of the IBC wasm contracts
+func IBCWasmCapabilities() []string {
```
Comment (SebastianElvis) [inline app/app.go:1333]: There is a variable `wasmCapabilities` at L176 in this file. Now we could remove that

**Review by SebastianElvis — COMMENTED** [review]
It's weird that e2e test in CI failed. Will take a look

File: app/app.go (hunk 1 above)
Comment (aeryz) [reply app/app.go]: Done

**Review by SebastianElvis — APPROVED** [review]
Thank you!

**Pull request merged**